files/leaflet.esri.related/update.json

```diff
@@ -0,0 +1,10 @@
+{
+  "packageManager": "github",
+  "name": "leaflet.esri.related",
+  "repo": "jgravois/esri-leaflet-related",
+  "files": {
+    "basePath": "dist/",
+    "include": ["esri-leaflet-related.js", "esri-leaflet-related.js.map"],
```

Comment:
Include only the min file and the related map file https://github.com/jgravois/esri-leaflet-related/tree/v0.0.1-beta.2/dist